src-graphql/image-query.js

```diff
@@ -0,0 +1,9 @@
+export default function imageQuery(fields = ['id', 'src', 'altText']) {
+  return function(parentQuery, fieldName) {
+    parentQuery.add(fieldName, (image) => {
```

Comment:
This is a small thing, but the parent isn't a query, it's a `selectionSet`. `parentSelection` is probably a more accurate name.